modules/homefinder/widgets/homefinder-widgets.js

```diff
@@ -0,0 +1,33 @@
+(function () {
+	hf_data.forEach(function(data, index) {
+		if ( data.type === 'directory' ) {
+			var widget = new HomeFinder.Widgets.AdvertiserDirectory({
+				container: 'directoryPreview-' + index,
+				data: {
+					profileName: data.affiliate,
+					maxCount: data.count
+				}
+			});
+		} else if ( data.type === 'search' ) {
+			var widget = new HomeFinder.Widgets.Search({
```

Comment:
__[jshint]__
* 'widget' is already defined.
* 'HomeFinder' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->